marklogic-data-hub-central/ui/src/components/entities/mapping/mapping-card.tsx

```diff
@@ -90,7 +90,7 @@ const MappingCard: React.FC<Props> = (props) => {
   useEffect(() => {
     let sortedArray = props.data.length > 1 ? sortStepsByUpdated(props.data) : props.data;
     setSortedMappings(sortedArray);
-    setSourceData([]);
+    // setSourceData([]);
```

Comment:
If this line of code isn't needed - it apparently is causing a problem - let's delete it! Commented-out code is generally an anti-pattern; we have branches and version history for storing code we're not ready to integrate yet.

Reply:
Thanks @rjrudin, removed the faulty line vs commenting it.